File: Labs/Lab-1/VP-1/src/vp/pkg1/Student.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package vp.pkg1;

/**
 *
 * @author Student
 */
public class Student extends Person{
    
    Student(){}
    
    Student(String name, int age, long phoneNumber, String email) {
        
        this.setName(name);
        this.setAge(age);
        this.setPhoneNumber(phoneNumber);
        this.setEmail(email);
        
    }
    
    @Override
    public void behaviour(){
        System.out.println("Paying fees!");
    }
    
}

File: Labs/Lab-2/Person/Main/Main/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Main {
    class Student : Person {

        Student() { }

        public Student(String name, int age, Int64 phoneNumber, String email)
        {

            this.setName(name);
            this.setAge(age);
            this.setPhoneNumber(phoneNumber);
            this.setEmail(email);

        }
        public void behaviour() {
            Console.WriteLine("Paying fees!");
        }

    }
}
